### PR TITLE
add ES function

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# Elasticsearch query functions
+
+The Elasticsearch query functions allow Nebula Graph to send a GET request to Elasticsearch to read the data and convert the returned JSON into a Map format in Nebula Graph, users can use the map data in the same composite graph query statement.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md:3]: . Users

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+    Only available for the Nebula Graph Enterprise Edition.
+
+## Precautions
+
+- You can not use the Elasticsearch service that a full-text index has been deployed. You need to use the independent Elasticsearch service and write data to the Elasticsearch service by yourself. Nebula Graph will not write data to the Elasticsearch. To deploy an Elasticsearch cluster, see [Kubernetes Elasticsearch deployment](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-deploy-elasticsearch.html) or [Elasticsearch installation](https://www.elastic.co/guide/en/elasticsearch/reference/7.15/targz.html).
+
+- `SIGN IN` is required before use the Elasticsearch query functions, for example, `SIGN IN TEXT SERVICE (127.0.0.1:9200, HTTP)`. You can `SIGN OUT` the service when it is no longer needed. For more Information, see  [Sign in to the text search clients](../../4.deployment-and-installation/6.deploy-text-based-index/2.deploy-es.md#_sign_in_to_the_text_search_clients).
+
+- The Elasticsearch query functions support only GET requests. Perform the POST, PUT, and DELETE requests using the Elasticsearch client.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md:17]: The POST, PUT ... should be called by the Elasticsearch client.

File: docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+- The Elasticsearch query functions have no timeout, Nebula Graph will wait for Elasticsearch to return results.
+
+- Using operator directly after the Elasticsearch query functions is not supported, sucha as `RETURN calles("/test/player/1")["_source"]`.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/6.functions-and-expressions/17.ES-function.md:21]: For example , ... . is not allowed.

**Review by whitewum — DISMISSED** [review]

File: mkdocs.yml [hunk 4]
```diff
@@ -300,6 +300,7 @@ nav:
           - concat(): 3.ngql-guide/6.functions-and-expressions/13.concat.md
           - Predicate functions: 3.ngql-guide/6.functions-and-expressions/8.predicate.md
           - Geography functions: 3.ngql-guide/6.functions-and-expressions/14.geo.md
+#          - Elasticsearch functions: 3.ngql-guide/6.functions-and-expressions/17.ES-function.md
```
Comment (abby-cyber) [inline mkdocs.yml:303]:
```suggestion
         - Elasticsearch functions: 3.ngql-guide/6.functions-and-expressions/17.ES-function.md
```

**Review by foesa-yang — APPROVED** [review]

**Pull request merged**